session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -916,23 +922,14 @@ public boolean checkTimeseriesExists(String path)
     return result;
   }
 
-  private synchronized String getTimeZone()
-      throws StatementExecutionException, IoTDBConnectionException {
+  private synchronized String getTimeZone() {
     if (zoneId != null) {
       return zoneId.toString();
     }
-
-    TSGetTimeZoneResp resp;
-    try {
-      resp = client.getTimeZone(sessionId);
-    } catch (TException e) {
-      throw new IoTDBConnectionException(e);
-    }
-    RpcUtils.verifySuccess(resp.getStatus());
-    return resp.getTimeZone();
+    return ZoneId.systemDefault().getId();
```

Comment:
There is something wrong with the logic. 
It should be the default value that the server stores when the client is not registered with the server.